requirements/app/docs.txt

```diff
@@ -1,17 +1,8 @@
-sphinx>=4.0,<5.0
-myst-parser>=0.15,<0.17
-nbsphinx>=0.8.5, <=0.8.9
+-r ../docs.txt
+
 ipython[notebook]
 ipython_genutils
-pandoc>=1.0, <=2.2
-docutils>=0.16, <0.19
-sphinxcontrib-fulltoc>=1.0, <=1.2.0
-sphinxcontrib-mockautodoc
+pytorch-lightning
```

Comment:
It is required for app docs?

Reply:
Yes, because docs pull-in examples and some examples are a training model...
otherwise, the docs would need to install all particular requirements from each example app